op-challenger/game/fault/register.go

```diff
@@ -23,7 +23,7 @@ import (
 
 var (
 	cannonGameType         = uint8(0)
-	outputCannonGameType   = uint8(0) // TODO(client-pod#260): Switch the output cannon game type to 1
+	outputCannonGameType   = uint8(253) // TODO(client-pod#260): Switch the output cannon game type to 1
```

Comment:
Is there another issue for switching this game type to `1`?

Reply:
Asking since this pr will close #260 and then this comment will be wrong

Reply:
Looks like this should just be a bullet under https://github.com/ethereum-optimism/client-pod/issues/43?
```suggestion
	outputCannonGameType   = uint8(253) // TODO(client-pod#43): Switch the output cannon game type to 1
```